src/07-registers/auxiliary/src/lib.rs

```diff
@@ -3,28 +3,38 @@
 #![deny(warnings)]
 #![no_std]
 
-#[allow(unused_extern_crates)] // NOTE(allow) bug rust-lang/rust#53964
 extern crate panic_itm; // panic handler
```

Comment:
```suggestion
use panic_itm as _; // panic handler
```
This fails on CI otherwise due to old idioms

Reply:
You're suggesting `use panic_item as _;` but that idiom isn't used anywhere else,
instead `pub use panic_itm;` is used:

```
$ find . -type f -name '*.rs' | xargs grep -i 'panic_itm'
./book/08-leds-again/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./book/07-registers/auxiliary/src/lib.rs://pub use panic_itm; // panic handler
./book/07-registers/auxiliary/src/lib.rs:use panic_itm as _; // panic handler
./book/05-led-roulette/auxiliary/src/lib.rs:pub use panic_itm; // panic handler
./book/06-hello-world/auxiliary/src/lib.rs:pub use panic_itm;
./book/14-i2c/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./book/09-clocks-and-timers/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./book/11-usart/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./book/15-led-compass/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./book/16-punch-o-meter/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./src/08-leds-again/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./src/07-registers/auxiliary/src/lib.rs:use panic_itm as _; // panic handler
./src/05-led-roulette/auxiliary/src/lib.rs:pub use panic_itm; // panic handler
./src/06-hello-world/auxiliary/src/lib.rs:pub use panic_itm;
./src/14-i2c/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./src/09-clocks-and-timers/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./src/11-usart/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./src/15-led-compass/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
./src/16-punch-o-meter/auxiliary/src/lib.rs:extern crate panic_itm; // panic handler
```
I assume I should change all:
```
#[allow(unused_extern_crates)] // NOTE(allow) bug rust-lang/rust#53964
extern crate panic_itm; // panic_handler
```
To either `pub use panic_itm;` or `use panic_itm as _;`?

Reply:
After reading [use-declarations](https://doc.rust-lang.org/reference/items/use-declarations.html) I'm going to go ahead with `use panic_itm as _;`.  It seems the best choice as there seems to be no reason to re-export by making it public, but we have to indicate that its the `panic_handler` we want. As I make modications moving forward I'll change all occurances to be `use panic_itm as _;`.